File: api/src/main/java/eu/suro/grpc/GRPCConfig.java
package eu.suro.grpc;

import de.exlll.configlib.Comment;
import de.exlll.configlib.Configuration;
import lombok.Data;
import lombok.Getter;

@Configuration
@Data
public class GRPCConfig {

    @Comment("Адрес сервера ядра")
    private String host = "localhost";
    @Comment("Порт сервера ядра")
    private int port = 9000;
}

File: api/src/main/kotlin/eu/suro/grpc/GRPCConfig.kt
package eu.suro.grpc

import de.exlll.configlib.Comment
import de.exlll.configlib.Configuration

@Configuration
class GRPCConfig {

    @Comment("Адрес сервера ядра")
    var host: String = "localhost"

    @Comment("Порт сервера ядра")
    var port = 9000;
}